composer.json

```diff
@@ -1,54 +1,54 @@
 {
-  "name": "subfission/cas",
-  "description": "Adds CAS support to Laravel 5.x - 10.x",
-  "keywords": [
-    "CAS",
-    "phpCAS",
-    "SSO",
-    "laravel",
-    "laravel 6",
-    "laravel 7",
-    "laravel 8",
-    "laravel 9"
-  ],
-  "license": "MIT",
-  "authors": [
-    {
-      "name": "subfission",
-      "role": "Developer"
-    }
-  ],
-  "require": {
-    "php": ">=7.2.0",
-    "illuminate/support": "^6.0|^7.0|^8.0|^9.0|^10.0",
-    "apereo/phpcas": "^1.6"
-  },
-  "require-dev": {
-    "phpunit/phpunit": "^8.0|^9.0|^10.0",
-    "pestphp/pest": "^1.21"
-  },
-  "autoload": {
-    "psr-4": {
-      "Subfission\\Cas\\": "src/Subfission/Cas/"
+    "name": "subfission/cas",
+    "description": "Adds CAS support to Laravel 5.x - 10.x",
```

Comment:
to 11.x though, right?

Reply:
Yes.  The bot wants require to also include a reference to 11.